requirements.txt

```diff
@@ -14,6 +14,7 @@ django_analytical>=2.5
 tqdm>=3.7.0
 wagtail-markdown
 Pygments
+wagtail-orderable
```

Comment:
**wagtail-orderable** seems to be not supported.
Reference: https://github.com/elton2048/wagtail-orderable/issues/30
Is there an alternative for this dependency?

Reply:
Not as far as I know.

This is one of those projects that seems to just be "complete" while being small enough to bugfix if there are future issues.

Reply:
In that case, let's keep this dependency.